src/workers/parser/utils/helpers.js

```diff
@@ -50,7 +51,7 @@ export function getObjectExpressionValue(node: Node) {
     return value.name;
   }
 
-  if (t.isCallExpression(value)) {
+  if (t.isCallExpression(value) || t.isFunctionExpression(value)) {
```

Comment:
this logic is a bit brittle... the idea is we can evaluate object values, if they are safe...

so for instance if the user hovers over `a` in `{a: b}` we'll evaluate `b`... the risk is that b might have side effects... so  we try and prevent that. I'm filtering by function here because it is not a common case and makes the snapshot look bad :) we generally wouldn't want to evaluate the render function either `{render: () => {}}`